test/unittests/mongoBackend/mongoNotifyContext_test.cpp

```diff
@@ -898,3 +900,77 @@ TEST(mongoNotifyContextRequest, createEntity)
     utExit();
 }
 
+
+
+/* ****************************************************************************
+*
+* templateNotificationVerb -
```

Comment:
This (and the next) are the "stimulation unit test" you are using for manual testing? I'd suggest to include a "FIXME: to be removed" as a reminder.